src/style.css

```diff
@@ -1 +1,99 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.page__body {
+  margin-top: 0;
+}
+
+.search__input {
+  box-sizing: border-box;
+  margin-top: 20px;
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+
+  color: #3d4e61;
+  font-family: Avenir, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 21.86px;
```

Comment:
If you see any value in pixels in float format always round it off, you can't write it that way